src/components/PostDetails.tsx

```diff
@@ -2,92 +2,31 @@ import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import {
+  deleteComment,
+  loadPostComments,
+} from '../features/comments/сommentsSlice';
```

Comment:
There seems to be a typo in the import path for `deleteComment` and `loadPostComments`. The folder name should be `commentsSlice` instead of `сommentsSlice` (note the difference in the first letter 'c'). Please correct this to avoid import errors.